test/AST-Quake/base_profile.cpp

```diff
@@ -29,9 +29,9 @@ struct kernel {
 
 // CHECK-LABEL: define void @__nvqpp__mlirgen__kernel
 // CHECK-SAME: ()
-// CHECK: tail call void @__quantum__qis__mz__body(%Qubit* null, %Result* null)
-// CHECK: %[[VAL_2:.*]] = tail call i1 @__quantum__qis__read_result__body(%Result* null)
-// CHECK: tail call void @__quantum__qis__mz__body(%Qubit* nonnull inttoptr (i64 1 to %Qubit*), %Result* nonnull inttoptr (i64 1 to %Result*))
-// CHECK: %[[VAL_3:.*]] = tail call i1 @__quantum__qis__read_result__body(%Result* nonnull inttoptr (i64 1 to %Result*))
+// CHECK: tail call void @__quantum__qis__mz__body(ptr null, ptr null)
```

Comment:
Yea this not what we want (yet, maybe in something the QIR Alliance takes up in the future).

Reply:
It looks like with LLVM 17, which is now the tip of llvm-project, this possibility is actually _not_ supported any longer. Setting opaque pointers to `false` just causes LLVM to have assertion errors.

Reply:
Opened an issue.